### ISSUE TITLE
Sha256 checksum error for tcllib 1.18

### ISSUE THREAD

**Issue by genovese** [Homebrew/homebrew-dupes#657]
Upgrade of the previous version of tcllib fails with:

```
Error: SHA256 mismatch
Expected: bcf0ba7656e3a99ffa5fc0ce0f4a95616530979539bddb8da43903b82983603f
Actual: 6a87881f545afb69c1130f60984b5d35cc22f1593b0835b982871c188fde3de8
Archive: /Users/***/Library/Caches/Homebrew/tcl-tk--tcllib-1.18.tar.gz
```

When I directly download the file at the given URL, it matches the latter (6a8788...),
not the one in the formula.

This is on OS X 10.12 (after a recent upgrade from 10.7).

**Comment by bhavinkamani** [comment]
I am facing the same issue

**Comment by kangstarr** [comment]
Same issue here as well. Looks like the SHA256 needs to get updated for "tcl-tk--tcllib-1.18.tar.gz"

**Comment by lvale** [comment]
Same here

```
==> Downloading https://github.com/tcltk/tcllib/archive/tcllib_1_18.tar.gz
==> Downloading from https://codeload.github.com/tcltk/tcllib/tar.gz/tcllib_1_18
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: bcf0ba7656e3a99ffa5fc0ce0f4a95616530979539bddb8da43903b82983603f
Actual: 6a87881f545afb69c1130f60984b5d35cc22f1593b0835b982871c188fde3de8
Archive: /Users/Luis/Library/Caches/Homebrew/tcl-tk--tcllib-1.18.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by apjanke** [comment]
Fixed by #659.
